packages/app/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import Image from "next/image";
+import Blocks from "../../public/assets/images/blocks.svg";
+import Link from "next/link";
+import MobileMenu from "./MobileMenu";
+import Logo from "./Logo";
+
+export default function Header() {
```

Comment:
Let's change the name of the component to `Navbar`
```suggestion
export default function Navbar() {
```